py/deploy.py

```diff
@@ -114,7 +114,7 @@ def test(args):
     start = time.time()
     util.run(["helm", "test", "tf-job"])
   except subprocess.CalledProcessError as e:
-    t.failure = "helm test failed;\n" + e.output
+    t.failure = "helm test failed;\n" + (e.output or '')
```

Comment:
use " not ' to be consistent with the rest of the file.

Reply:
@jlewi Sorry I don't follow. I don't see `not` in the rest of the file 
Do you mean like an if else statement?
```
if not e:
  t.failure = "helm test failed;\n"
else:
  t.failure = "helm test failed;\n" + e.output
```

Reply:
I was referring to the quotation marks. We are using double quotes not single quotes in this file so we should be consistent.

Reply:
Ah got it

Reply:
@jlewi Ok this should be addressed